components/engine_traits/Cargo.toml

```diff
@@ -11,5 +11,6 @@ protobuf = "2"
 quick-error = "1.2.3"
 tikv_alloc = { path = "../tikv_alloc" }
 tikv_util = { path = "../tikv_util" }
+txn_types = { path = "../txn_types" }
```

Comment:
I would rather not add txn_types as a dependency, since it depends on kvproto. Conceptually, there's nothing about storage engines that _should_ depend on grpc definitions. Anything grpc related belongs at a higher level of abstraction. Is there any way to get rid of the kvproto dependency?

Reply:
😂 Seems `RaftEngine` has introduced kvproto dependency, cannot get rid of it for now.